README.md

```diff
@@ -51,16 +51,12 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://www.loom.com/share/7f24ada1fc6f40868fc4c9a6e934fb7b)
+- [Live Demo Link](https://www.loom.com/share/c6adcc8020ae40c9a7d64e3bc3f9a937)
```

Comment:
As I requested before, please add the deployed project live link to your README file. Currently, there is a Loom video.  So it will look like this below:
```
- [Live Demo Link]
(https://innocent259.github.io/My-portfolio/)
```